internal/services/device_definitions_service.go

```diff
@@ -406,17 +410,30 @@ func (d *deviceDefinitionService) PullDrivlyData(ctx context.Context, userDevice
 	}
 	localLog := d.log.With().Str("vin", vin).Str("deviceDefinitionID", deviceDefinitionID).Logger()
 
-	neverPulledVIN := false
+	client, conn, err := d.getVINDecodeGrpcClient()
+
+	if err != nil {
+		return ErrorDataPullStatus, errors.Wrap(err, "error getting grpc client")
+	}
+
+	defer conn.Close()
+
+	_, err = client.DecodeVin(ctx, &ddgrpc.DecodeVinRequest{Vin: vin})
+
+	if err != nil {
+		return ErrorDataPullStatus, errors.Wrap(err, "error pulling vin info from drivly")
+	}
+
 	existingVINData, err := models.ExternalVinData(
```

Comment:
i think we may not need this anymore, since we never want to pull VIN data anymore from here, since it will all be handled in the vin decode in device-definitions-api. Could probably also remove the forceSetVinInfo since that metadata is handled also with the decode vin